MapboxVisionTests/RecordingQuotaTests.swift

```diff
@@ -0,0 +1,109 @@
+@testable import MapboxVision
+import XCTest
+
+class RecordingQuotaTests: XCTestCase {
+    // MARK: - Private properties
+
+    private var refreshInterval: TimeInterval!
+    private var initialMemoryQuota: Byte!
+    private var recordingQuota: RecordingQuota!
+
+    // MARK: - Test functions
+
+    override func setUp() {
+        super.setUp()
+        refreshInterval = TimeInterval(Int.random(in: 1...5))
```

Comment:
As discussed, random time doesn't provide much value here as there're no changing environment factors to it.
Also as discussed there's no reason to make wait time more than a second.